### DISCUSSION TITLE
Integrate with NeoVim distro

### DISCUSSION THREAD

**Discussion by bengidev** [mickael-menu/ShadowVim#64] [Help]
### Verify

- [X] I searched the existing discussions for help

### How can we help you?

Hello, is this plugins can integrate with NeoVim distro, like AstroVim, LazyVim, or NvChad?

And, is this plugins support code completions or code suggestions?

Thank you for your great work 🍻

**Comment by mickael-menu** [comment]
I don't know, but unless they mess up with the Neovim starting process, it should work fine.

Personally I'm using `lazy.nvim` to manage my plugins with ShadowVim.

I recommend starting from an empty nvim config and try enabling things progressively from there, see https://github.com/mickael-menu/ShadowVim/tree/develop#neovim-configuration.

As for the code completion and code suggestions, it is still the ones from Xcode that are used while in insert mode.
